extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/sql/resources/SqlStatementResource.java

```diff
@@ -735,8 +735,8 @@ private Optional<Yielder<Object[]>> getResultYielder(
         );
       }
 
-      MSQTaskReportPayload msqTaskReportPayload = jsonMapper.convertValue(SqlStatementResourceHelper.getPayload(
-          contactOverlord(overlordClient.taskReportAsMap(queryId), queryId)), MSQTaskReportPayload.class);
+      MSQTaskReportPayload msqTaskReportPayload = SqlStatementResourceHelper.getPayload(
+          contactOverlord(overlordClient.taskReportAsMap(queryId), queryId));
```

Comment:
nit: for readability
```suggestion
      MSQTaskReportPayload msqTaskReportPayload = SqlStatementResourceHelper.getPayload(
          contactOverlord(overlordClient.taskReportAsMap(queryId), queryId)
      );
```